doc/fbchkdoc/fmtcode.bas

```diff
@@ -135,17 +136,31 @@ private function _emitCode ( byref text as string ) as string
 
 end function
 
-'' main entry point
-function FormatFbCode( byref txt as string ) as string
-	
+''
+function FormatFbCodeLoadKeywords( byref filename as string ) as boolean
+
 	static kw_loaded as integer = FALSE
 
 	if( kw_loaded = FALSE ) then
 		'' Load the keywords, this will allow keywords to be cased.
-		fbdoc_loadkeywords( "../manual/templates/default/keywords.lst" )
+
+		if( filename > "" ) then
```

Comment:
How about rewriting that as:
```
if( filename = "" ) then
    filename = default_ManualDir + "templates/default/keywords.lst"
end if
function = ( fbdoc_loadkeywords( filename ) <> 0 )
```

Reply:
I like your suggestion, it flows better.  It also highlights that I wrote most of this code before const qualifiers and will have to revisit the code base for an overall update.

Reply:
Hmm, yea, modifying the Byref argument may be unexpected, so a copy is needed. However, as far as I saw, FormatFbCodeLoadKeywords() is never called with an empty string at the moment, so it may not even need to handle the case.